ping @crosbymichael; got a golint error on Janky;

```
00:35:10 Errors from golint:
00:35:10 restartmanager/restartmanager.go:17:5: exported var ErrRestartCanceled should have comment or be unexported
00:35:10 
00:35:10 Please fix the above errors. You can test via "golint" and commit the result.
```

--- a/libcontainerd/container_linux.go
+++ b/libcontainerd/container_linux.go
@@ -9,6 +9,7 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	containerd "github.com/docker/containerd/api/grpc/types"
+	"github.com/docker/docker/restartmanager"
 	"github.com/opencontainers/specs/specs-go"
 	"golang.org/x/net/context"
 )
@@ -135,7 +136,9 @@ func (ctr *container) handleEvent(e *containerd.Event) error {
 								logrus.Error(err)
 							}
 						})
-						logrus.Error(err)
+						if err != restartmanager.ErrRestartCanceled {
+							logrus.Error(err)
+						}
 					} else {
 						ctr.start()
 					}

--- a/restartmanager/restartmanager.go
+++ b/restartmanager/restartmanager.go
@@ -1,6 +1,7 @@
 package restartmanager
 
 import (
+	"errors"
 	"fmt"
 	"sync"
 	"time"
@@ -12,6 +13,10 @@ const (
 	backoffMultiplier = 2
 	defaultTimeout    = 100 * time.Millisecond
 )
+
+// ErrRestartCanceled is returned when the restart manager has been
+// canceled and will no longer restart the container.
+var ErrRestartCanceled = errors.New("restart canceled")
 
 // RestartManager defines object that controls container restarting rules.
 type RestartManager interface {
@@ -54,7 +59,7 @@ func (rm *restartManager) ShouldRestart(exitCode uint32, hasBeenManuallyStopped 
 	}()
 
 	if rm.canceled {
-		return false, nil, fmt.Errorf("restartmanager canceled")
+		return false, nil, ErrRestartCanceled
 	}
 
 	if rm.active {
@@ -95,7 +100,7 @@ func (rm *restartManager) ShouldRestart(exitCode uint32, hasBeenManuallyStopped 
 	go func() {
 		select {
 		case <-rm.cancel:
-			ch <- fmt.Errorf("restartmanager canceled")
+			ch <- ErrRestartCanceled
 			close(ch)
 		case <-time.After(rm.timeout):
 			rm.Lock()

